### PR TITLE
Fixing sample scripts

### PR DESCRIPTION
**Pull request by vinaykumarvvs-lt**
Fixed the failed tests
Updated the PW to latest version
Updated the Sample Website to DuckDuckGo Instead of Bing
Fixed the ReadMe & Build Names

### CODE REVIEW

**Review by ankur-lt — CHANGES_REQUESTED** [review]
LGTM. @vinaykumarvvs-lt , please add CI config for all test suites

**Comment by ankur-lt** [comment]
@vinaykumarvvs-lt , please raise the PR first on develop branch, then we can merge it on main.

**Comment by ankur-lt** [comment]
Changed the base to develop.

File: .gitignore [hunk 1]
```diff
@@ -109,3 +109,8 @@ dist
 
 **/target/
 **/.DS_Store
+
+**/bin/
+**/obj/
+
+**/*.feature.cs
```
Comment (ankur-lt) [inline .gitignore:116]: Why is `**/*.feature.cs` added to .gitignore?

**Review by ankur-lt — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (vinaykumarvvs-lt) [reply .gitignore:116]: @ankur-lt on compiling the dotnet package it generates the .cs files. That's why added in the gitignore

**Review by ankur-lt — APPROVED** [review]
LGTM ✅
CI checks to be added in next PR.

**Pull request merged**